repotracker/repotracker.go

```diff
@@ -817,6 +812,15 @@ func createVersionItems(ctx context.Context, v *model.Version, ref *model.Projec
 
 		_, err = evergreen.GetEnvironment().DB().Collection(model.VersionCollection).InsertOne(sessCtx, v)
 		if err != nil {
+			cmdErr, isCmdErr := err.(mongo.CommandError)
```

Comment:
why can't we use "isTransientTxError" here?

Reply:
oops, fixed